src/PrestaShopBundle/Twig/Layout/MenuBuilder.php

```diff
@@ -63,12 +70,33 @@ public function getCurrentTab(): ?Tab
             }
         }
 
+        $this->currentTab = $tab;
+
         return $tab;
     }
 
+    public function getCurrentTabLevel(): int
+    {
+        $currentTab = $this->currentTab ?: $this->getCurrentTab();
```

Comment:
```suggestion
        $currentTab = $this->getCurrentTab();
```
Since `getCurrentTab` already checks if the local field is present I think you can simplify this code